java/core/src/java/org/apache/orc/DataReader.java

```diff
@@ -55,8 +55,14 @@ BufferChunkList readFileData(BufferChunkList range,
    * Releases buffers created by readFileData. See readFileData javadoc.
    * @param toRelease The buffer to release.
    */
+  @Deprecated
```

Comment:
In order to deprecate properly, we need additional Javadoc message with the alternative usage, e.g., `Use ... instead`. Please see the example.

https://github.com/apache/orc/blob/3d9aa415d28d5f9d21d389018441cb499983f2d8/java/core/src/java/org/apache/orc/OrcConf.java#L306-L313